book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(rental)
+    @rental.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] This method suppose to add a new rental and the book itself should be sent as a parameter achieving the 'has-many' association. So please fix that.
[Hint] You can do it like that:
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```